mobile/lib/utils/file_uploader.dart

```diff
@@ -469,6 +472,16 @@ class FileUploader {
     }
   }
 
+  void _uploadPreview(EnteFile file) {
```

Comment:
This should only be called for internal users.

Reply:
Resolved!